app/views/CreateChannelView.js

```diff
@@ -69,11 +69,12 @@ const styles = StyleSheet.create({
 
 class CreateChannelView extends React.Component {
 	static navigationOptions = () => ({
-		title: I18n.t('Create_Channel')
-	});
+		title: this.isTeam ? I18n.t('Create_Team') : I18n.t('Create_Channel')
```

Comment:
It doesn't look correct.
How are you accessing `this` inside a `static`?
Maybe a `this.setHeader` would be better.